support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/serializer/JacksonBinarySerializer.java

```diff
@@ -0,0 +1,81 @@
+package org.apereo.cas.serializer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.apereo.cas.TicketSerializer;
+import org.apereo.cas.ticket.ServiceTicket;
+import org.apereo.cas.ticket.ServiceTicketImpl;
+import org.apereo.cas.ticket.Ticket;
+import org.apereo.cas.ticket.TicketGrantingTicket;
+import org.apereo.cas.ticket.TicketGrantingTicketImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.1.0
+ */
+public class JacksonBinarySerializer implements TicketSerializer<ByteBuffer> {
```

Comment:
We would like to be able to use different serializers, you could plugin on the configuration class (now that I have extracted that bean as you suggested)